filebeat/input/log/input.go

```diff
@@ -78,6 +79,8 @@ type Input struct {
 	meta                map[string]string
 	stopOnce            sync.Once
 	fileStateIdentifier file.StateIdentifier
+	getStatusReporter   input.GetStatusReporter
+	healthy             bool
```

Comment:
You should not need this flag. This state is already persisted in the agent, you just need to directly send status updates when it runs, fails, or stops.

Reply:
Thanks for pointing this out. I've removed it